PhoneNumberKit/UI/TextField.swift

```diff
@@ -88,6 +88,14 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
         }
     }
 
```

Comment:
wouldn't it be better to add `@available(iOS 11, *)` here instead of the `didSet` since nothing will happen for lower versions anyway if this is set to `true`

Reply:
Because it's a stored property you can't actually annotate with `@available`, I will work around this by doing the following, let me know if you prefer this. It does at least make the caller aware that it will only work on iOS >11
```
private var _withDefaultPickerUI: Bool = false {
    didSet {
        if #available(iOS 11.0, *), flagButton.actions(forTarget: self, forControlEvent: .touchUpInside) == nil {
            flagButton.addTarget(self, action: #selector(didPressFlagButton), for: .touchUpInside)
        }
    }
}

@available(iOS 11.0, *)
public var withDefaultPickerUI: Bool {
    get { _withDefaultPickerUI }
    set { _withDefaultPickerUI = newValue }
}
```

Reply:
This looks better, sorry for the late reply I've been busy with work.